README.md

```diff
@@ -131,6 +131,8 @@ These settings are specific to VSCodeVim.
 | vim.searchHighlightTextColor     | Foreground color of non-current search matches                                                                                                                                                                                                                                                                                                                                                                                      | String  | None                                      |
 | vim.searchMatchColor             | Background color of current search match                                                                                                                                                                                                                                                                                                                                                                                            | String  | `findMatchBackground` ThemeColor          |
 | vim.searchMatchTextColor         | Foreground color of current search match                                                                                                                                                                                                                                                                                                                                                                                            | String  | None                                      |
+| vim.substitutionColor            | Background color of substituted text when `vim.inccommand` is enabled                                                                                                                                                                                                                                                                                                                                                               | String  | "#50f01080"                               |
```

Comment:
```suggestion
| vim.substitutionColor            | Background color of substitution text when `vim.inccommand` is enabled                                                                                                                                                                                                                                                                                                                                                               | String  | "#50f01080"                               |
```

Maybe? This is the text that is doing the substituting, not being substituted, right?